Source/iOS/Device.swift

```diff
@@ -191,6 +193,8 @@ open class Device {
                 return .screen10_9Inch
             case 1194:
                 return .screen11Inch
+            case 1376:
```

Comment:
@divyesh-makwana : Can you please put 1376 below 1366? thanks. Rest all okay.

Reply:
Done!!